assets/dev/js/editor/document/elements/commands/paste-style.js

```diff
@@ -9,6 +9,8 @@ export class PasteStyle extends History {
 		let result = true;
 
 		if (
+			null === source ||
+			null === target ||
 			undefined === source ||
 			undefined === target ||
```

Comment:
I'm pretty sure the `_.empty` Underscore's method is handling this case

Reply:
Will be fixed in next document-commands branch.